src/pages/Donate/Donate.js

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { useDispatch } from 'react-redux';
 import { openModal } from './../../actions/index';
 
+import DonateSectionOne from './../../sections/Donate/DonateSectionOne/DonateSectionOne';
 import CustomThankYou from './../../components/Custom/CustomThankYou/CustomThankYou';
 
-const Donate = () => {
+
+const Donate = ({ data }) => {
   const dispatch = useDispatch();
   const thankYou = <CustomThankYou />
 
   return (
     <div>
-      <h4>Donate</h4>
+      { data ? <DonateSectionOne data={data.section_1} /> : null }
```

Comment:
you'll probably need this for more than just this component - instead you should wrap the return in an if statement

Reply:
can you take a swing at optimizing the app so we don't need todo this?